core/services/ocr2/delegate.go

```diff
@@ -307,6 +307,10 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 			return nil, errors.Wrap(err2, "get chainset")
 		}
 
```

Comment:
We still need this to do the following:
1- default sendingKeys to transmitterId for all other jobs (possibly above where the forwarder is set)
2- override this default behavior here for VRF to get all keys

The flow of rotating on all local keys through a forwarder is not /yet/ the default behavior for ocr2 jobs

Reply:
SGTM, made the needed change.